test/integration/licenses/cloud.google.com/go/compute/metadata/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
@NimishKashyap there are many unrelated files that were checked in please remove them and also ensure it is not in the commit history

Reply:
I have deleted those files. Please take a look into it.